app/src/main/kotlin/com/wire/android/util/lifecycle/ConnectionPolicyManager.kt

```diff
@@ -125,39 +124,25 @@ class ConnectionPolicyManager @Inject constructor(
     private suspend fun UserSessionScope.downgradePolicyIfNeeded(
         userId: UserId
     ) {
-        val isCurrentSession = isCurrentSession(userId)
         val isAppOnForeground = currentScreenManager.isAppOnForegroundFlow().first()
-        logger.d("isCurrentSession = $isCurrentSession; hasInitialisedUI = $isCurrentSession")
-        val shouldKeepLivePolicy = isCurrentSession && isAppOnForeground
-        if (!shouldKeepLivePolicy) {
-            logger.d("$TAG Downgrading policy as conditions to KEEP_ALIVE are not met")
+        logger.d("$TAG hasInitialisedUI = $isAppOnForeground")
+        if (!isAppOnForeground) {
+            logger.d("$TAG $userId Downgrading policy as conditions to KEEP_ALIVE are not met")
```

Comment:
Is it possible to obfuscate `$userId` here?

Reply:
Yup I forgot to hide it after debugging 🤦